lib/features/crew/presentation/crews_screen.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:galaxyxgate/core/themes/app_colors.dart';
+import 'package:galaxyxgate/core/themes/text_styles.dart';
+import 'package:galaxyxgate/features/crew/business_logic/cubit/crews_cubit.dart';
+import 'package:galaxyxgate/features/crew/data/models/crew_model.dart';
+import 'package:galaxyxgate/features/crew/presentation/widget/crew_grid.dart';
+import 'package:galaxyxgate/features/crew/presentation/widget/loader.dart';
+import 'package:flutter_staggered_animations/flutter_staggered_animations.dart';
+
+class CrewsScreen extends StatefulWidget {
+  const CrewsScreen({super.key});
+  @override
+  State<CrewsScreen> createState() => _CrewsScreenState();
+}
+
+class _CrewsScreenState extends State<CrewsScreen> {
+  initState() {
+    super.initState();
+    BlocProvider.of<CrewsCubit>(context).getCrews();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: CustomScrollView(
+            physics: const BouncingScrollPhysics(
+              parent: AlwaysScrollableScrollPhysics(),
+            ),
+            slivers: [
+              SliverAppBar(
+                backgroundColor: AppColors.lightBlack,
+                pinned: true,
+                expandedHeight: 50.h,
+                leading: IconButton(
+                    onPressed: () {},
+                    icon: const Icon(
+                      Icons.arrow_back_ios_new,
+                      color: AppColors.white,
+                    )),
+                title: const Text('Crews'),
+                titleTextStyle: TextStyles.font18White700w
+                    .copyWith(fontWeight: FontWeight.normal),
+              ),
+              BlocConsumer<CrewsCubit, CrewsState>(
```

Comment:
Why consumer ?
Consider making builder

Reply:
Because in the code I need to deal with the error state by displaying a SnackBar to the user and updating the UI based on the state (loadind / loaded), it is better to use BlocConsumer.
If I try to use BlocBuilder and add the error status, it will be called frequently. This could lead to the 'SnackBar' being shown repeatedly or inconsistently, confusing user experiences or unintended behavior 

The listener function in BlocConsumer is the appropriate place for managing these side effects without impacting the builder function, which should remain focused on rendering the UI according to the state.